data/clients.json

```diff
@@ -321,7 +323,7 @@
     },
     {
         "doap": null,
-        "last_renewed": "2018-08-29T15:00:00",
+        "last_renewed": "2021-05-20T18:25:00",
```

Comment:
I'm pretty sure it is not developed anymore: https://github.com/jitsi/jitsi/releases/tag/Jitsi-2.10